Source/Net.swift

```diff
@@ -17,6 +17,14 @@ public class Net {
     var sinkNodes = [NetNode]()
     var nodes = [NetNode]()
 
+    static internal var buildNet = Net()
```

Comment:
Let use a stack here. Nested calls to `Net.build` are unlikely but at least it won't give unexpected results if someone ever does it.